src/engraving/style/style.cpp

```diff
@@ -328,6 +328,8 @@ void MStyle::read(XmlReader& e, compat::ReadChordListHook* readChordListHook)
             set(Sid::dontHideStavesInFirstSystem, e.readBool());
         } else if (tag == "beamDistance") { // beamDistance maps to useWideBeams in 4.0
             set(Sid::useWideBeams, e.readDouble() > 0.75);
+        } else if (tag == "pedalListStyle") { // pre-3.6.3/4.0 typo
+            set(Sid::pedalLineStyle, e.readInt());
```

Comment:
I think it should nowadays be:
```suggestion
            set(Sid::pedalLineStyle, TConv::fromXml(e.readAsciiText(), LineType::SOLID));
```
That will probably make the VTests happy.

Reply:
Hmm, that is very different from the other special cases above this one?!?